From 3e9e78fc7e335d60aec11d7ac93ff7c343343a86 Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Mon, 19 Mar 2018 10:41:28 +0100
Subject: [PATCH] Increase sleep time in testsuite to make it more robust

This helps on loaded machines (and such fix the build on s390 in
openSUSE).
Thanks to R. Oertel for providing the fix.

Change-Id: Ia6b9c8f32e2a9d5c2cec826ec26959fba125e181
---
 oslo_log/tests/unit/test_log.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oslo_log/tests/unit/test_log.py b/oslo_log/tests/unit/test_log.py
index 553ad10..669081c 100644
--- a/oslo_log/tests/unit/test_log.py
+++ b/oslo_log/tests/unit/test_log.py
@@ -1355,7 +1355,7 @@ class MutateTestCase(BaseTestCase):
         mock_fileConfig.reset_mock()
 
         # No thread sync going on here, just ensure the mtimes are different
-        time.sleep(0.1)
+        time.sleep(1)
         os.utime(logini, None)
         self.CONF.mutate_config_files()
 
-- 
2.16.2

